R/task_bulk.R

```diff
@@ -83,7 +83,7 @@ bulk_prepare_expression_X <- function(X, do_call, use_names) {
       lens <- lengths(X)
       ## Here, support recycling out scalars
       ul <- unique(lens)
-      if (ul == 2L && min(ul) == 1L) {
+      if (length(ul) == 2L && min(ul) == 1L) {
```

Comment:
I'm a little unclear on why this change is necessary. Just an opportunistic fix or a change required for the PR?

Reply:
There was a logical comparison of nonscalars here if there were more than two elements - so this worked sometimes but only by accident. So yes, opportunistic fix, but triggered by updates in R CMD check that now screen for this

Reply:
Great, will merge then.